sphinx/templates/quickstart/conf.py_t

```diff
@@ -112,12 +112,16 @@ html_static_path = ['{{ dot }}static']
 #
 # This is required for the alabaster theme
 # refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
-html_sidebars = {
-    '**': [
-        'relations.html',  # needs 'show_related': True theme option to display
-        'searchbox.html',
-    ]
-}
+if html_theme == 'alabaster':
```

Comment:
this does not work if user uses `-D html_theme=alabaster` from command line.
It also does not work if user updates `html_theme` variable after this block.